src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx

```diff
@@ -40,6 +41,18 @@ export default function PaginatedIntegrationsList({
 		setCurrentPage(1)
 	}, [sortedIntegrations])
 
+	/**
+	 * If our pagination page changes, scroll up to the top of the wrapper
+	 * However, don't scroll to the wrapper if this is the first render.
+	 */
+	useEffect(() => {
+		if (isFirstRender.current) {
+			isFirstRender.current = false
+		} else {
+			window.scrollTo(0, 0)
+		}
+	}, [currentPage])
```

Comment:
Can we rely on the `onPageChange` callback from pagination vs needing to detect first render?